components/gfx/platform/macos/font_template.rs

```diff
@@ -34,18 +35,20 @@ pub struct FontTemplateData {
     ctfont: CachedCTFont,
 
     pub identifier: Atom,
-    pub font_data: Option<Vec<u8>>
+    pub font_data: Option<IpcSharedMemory>,
```

Comment:
I wonder if this should be `RefCell<Option<IpcSharedMemory>>` to memoize it when recreating the bytes from the `CachedCTFont` in the `bytes` method.